index.html

```diff
@@ -167,49 +167,100 @@ <h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
-					<li class="about-me__tag">Html</li>
-					<li class="about-me__tag">css</li>
-				  </ul>
+					<img class="about-me__icon" src="images/icon-languages.svg" alt="languages icon">
+					<div class="about-me__languages">Languages</div>
+					<ul class="about-me__tags">
+						<li class="about-me__tag">JavaScript</li>
+						<li class="about-me__tag">Ruby</li>
+						<li class="about-me__tag">Html</li>
+						<li class="about-me__tag">css</li>
+					</ul>
 				</li>      
 				<li class="about-me__works">
-				  <img class="about-me__icon" src="images/frameworks.svg" alt="languages icon">
-				  <div class="about-me__languages">Frameworks</div>
-				  <ul class="about-me__tags">
-					<li class="about-me__tag">Bootstrap</li>
-					<li class="about-me__tag">Ruby on rails</li>
-					<li class="about-me__tag">RSpec</li>
-					<li class="about-me__tag">Capybara</li>
-					<li class="about-me__tag">Selenium</li>
-				  </ul>
+					<img class="about-me__icon" src="images/frameworks.svg" alt="languages icon">
+					<div class="about-me__languages">Frameworks</div>
+					<ul class="about-me__tags">
+						<li class="about-me__tag">Bootstrap</li>
+						<li class="about-me__tag">Ruby on rails</li>
+						<li class="about-me__tag">RSpec</li>
+						<li class="about-me__tag">Capybara</li>
+						<li class="about-me__tag">Selenium</li>
+					</ul>
 				</li>
 				<li class="about-me__works">
-				  <img class="about-me__icon" src="images/skills.svg" alt="languages icon">
-				  <div class="about-me__languages">Skills</div>
-				  <ul class="about-me__tags">
-					<li class="about-me__tag">Codekit</li>
-					<li class="about-me__tag">GitHub</li>
-					<li class="about-me__tag">Codepen</li>
-					<li class="about-me__tag">Gitlab</li>
-					<li class="about-me__tag">Terminal</li>
-				  </ul>
+					<img class="about-me__icon" src="images/skills.svg" alt="languages icon">
+					<div class="about-me__languages">Skills</div>
+					<ul class="about-me__tags">
+						<li class="about-me__tag">Codekit</li>
+						<li class="about-me__tag">GitHub</li>
+						<li class="about-me__tag">Codepen</li>
+						<li class="about-me__tag">Gitlab</li>
+						<li class="about-me__tag">Terminal</li>
+					</ul>
 				</li>
-			  </ul>
+			</ul>
+		</section>
+		<!-- Contact Form Section -->
+		<section id="contact-form">			
+			<form class="contact-form__form" action="https://formspree.io/f/mzbqrajl" method="POST">
+				<div class="contact-form__primary">
+					<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+				</div>
+				<label for="name">
+					<input type="text" name="name" id="name" autocomplete="name" placeholder="Full Name" maxlength="30" class="contact-form__input" required>
+				</label>
+				<label for="email">
+					<input type="text" name="email" id="email" autocomplete="email" placeholder="Email address" class="contact-form__input" required>
+				</label>
+				<label for="textarea">
```

Comment:
- [ ] While you did a good job using `textarea` for the text field :+1:, I think it would be better if you add a `placeholder='Enter text here'` attribute instead of using a value 'Enter text here' because it would improve the User experience